mofacts/client/views/experiment/card.js

```diff
@@ -1775,7 +1775,7 @@ async function unitIsFinished(reason) {
   Session.set('curUnitInstructionsSeen', false);
 
   let leaveTarget;
-  if (newUnitNum < curTdf.tdfs.tutor.unit.length) {
+  if (newUnitNum < curTdf.tdfs.tutor.unit.length && typeof curTdfUnit.unitinstructions !== 'undefined'){
```

Comment:
This looks to be having an unintended effect where it will kick the user out of the session in-between units